scripts/screen_record.sh

```diff
@@ -54,15 +49,17 @@ stopRecording() {
 
     pkill -SIGINT -f wf-recorder
 
-    recentFile=$(ls -t "$outputDir"/recording_*.mp4 | head -n 1)
+    recentFile=$(ls -t "$tempDir"/*.mp4 | head -n 1)
 
-    notify-send "Recording stopped" "Your recording has been saved." \
-        -i video-x-generic \
-        -a "Screen Recorder" \
-        -t 10000 \
-        -u normal \
-        --action="scriptAction:-xdg-open $outputDir=Directory" \
-        --action="scriptAction:-xdg-open $recentFile=Play"
+    # Let the user pick a directory using Zenity
+    saveDir=$(zenity --file-selection --directory --title="Select Directory to Save Recording")
+    
+    if [ -n "$saveDir" ]; then
+        mv "$recentFile" "$saveDir/"
+        notify-send "Recording saved" "File saved to $saveDir."
```

Comment:
Looking at the code this replaces, its missing the action button to open the file location where it's saved, which I think is a nice addition to keep